src/main/java/uk/ac/cam/cl/dtg/segue/quiz/ChoiceQuestionValidator.java

```diff
@@ -41,8 +41,11 @@ public final QuestionValidationResponse validateQuestionResponse(final Question
         Validate.notNull(question);
         Validate.notNull(answer);
 
-        // check that the question is of type ChoiceQuestion before we go ahead
         ChoiceQuestion choiceQuestion;
+        // These variables store the important features of the response we'll send.
+        Content feedback = null;                        // The feedback we send the user
+        boolean responseCorrect = false;                // Whether we're right or wrong
+        // check that the question is of type ChoiceQuestion before we go ahead
         if (question instanceof ChoiceQuestion) {
```

Comment:
¯\_(ツ)_/¯
They're just boilerplate that exist in every single other validator for the same two variables. Since it has become the de facto way we write validators, they probably aren't so useful any more, especially since IntelliJ underlines them to show that they are widely used and part of the return value.